package.json

```diff
@@ -46,7 +46,7 @@
   "author": "Google Inc.",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=0.12"
+    "node": ">=4"
```

Comment:
Strictly speaking this is semver-major. Please include rationale for why this can be done without a major bump.